jsonb-platform-tck/pom.xml

```
@@ -0,0 +1,375 @@
+                                            <artifactId>arquillian-protocol-lib</artifactId>
+                                            <type>jar</type>
+                                            <overWrite>true</overWrite>
+                                            <outputDirectory>${project.build.directory}/protocol</outputDirectory>
+                                            <destFileName>protocol.jar</destFileName>
+                                        </artifactItem>
+                                    </artifactItems>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+
+                    <plugin>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <executions>
+
+                            <execution>
+                                <id>cdi-tests-appclient</id>
+                                <goals>
+                                    <goal>integration-test</goal>
+                                    <goal>verify</goal>
+                                </goals>
+                                <configuration>
+                                    <additionalClasspathElements>
+                                        <additionalClasspathElement>${project.build.directory}${file.separator}${payara.home}${file.separator}glassfish${file.separator}modules${file.separator}yasson.jar</additionalClasspathElement>
+                                        <additionalClasspathElement>${project.build.directory}${file.separator}${payara.home}${file.separator}glassfish${file.separator}modules${file.separator}parsson.jar</additionalClasspathElement>
+                                    </additionalClasspathElements>
+                                    <includes>
+                                        <include>com/sun/ts/tests/jsonb/cdi/**/*Ejb*.java</include>
+                                    </includes>
+                                    <!-- Select the @Tag("tck-appclient") tests -->
+                                    <groups>tck-appclient</groups>
+                                    <dependenciesToScan>jakarta.tck:jsonb-platform-tck</dependenciesToScan>
+                                    <systemPropertyVariables>
+                                        <payara.home>${payara.home}</payara.home>
+                                        <junit.log.traceflag>true</junit.log.traceflag>
+                                        <harness.log.traceflag>true</harness.log.traceflag>
+                                        <cts.harness.debug>true</cts.harness.debug>
+                                        <java.io.tmpdir>${project.build.directory}${file.separator}tmp</java.io.tmpdir>
+                                        <project.basedir>${project.basedir}</project.basedir>
+                                        <arquillian.xml>appclient-arquillian.xml</arquillian.xml>
```

Missing ts.home property

```suggestion
                                        <arquillian.xml>appclient-arquillian.xml</arquillian.xml>
                                        <ts.home>${ts.home}</ts.home>
```